Tophat/Utilities/RemoteControlReceiver.swift

```diff
@@ -90,5 +100,57 @@ struct RemoteControlReceiver {
 				)
 			}
 		}
+
+		Task {
```

Comment:
I need to better isolate responsibilities relating `RemoteControlReceiver` but coupling is pretty loose here, so I'll start with passing the dependencies in and handling here.

The delegate needs to be revised here.